server/api/crud/artifacts.py

```diff
@@ -144,8 +145,8 @@ def list_artifacts(
         self,
         db_session: sqlalchemy.orm.Session,
         project: str = None,
-        name: str = "",
-        tag: str = "",
+        name: typing.Optional[str] = "",
+        tag: typing.Optional[str] = "",
```

Comment:
it's either
```suggestion
        name: typing.Optional[str] = None,
        tag: typing.Optional[str] = None,
```
or as it was before